### PR TITLE
[bitnami/rabbitmq]fix secure password env hardcode

### PR DESCRIPTION
**Pull request by cleverhu**
fix secure password env hardcode.
Fixes: https://github.com/bitnami/charts/issues/12924
### Applicable issues

  - fixes https://github.com/bitnami/charts/issues/12924

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi!

Thank you so much for the PR! Could you fix the merge issue?

**Comment by cleverhu** [comment]
> Hi!
> 
> Thank you so much for the PR! Could you fix the merge issue?

I have updated the version./PTAL

**Comment by cleverhu** [comment]
The pipeline is failed, could anybody help to fix it?

**Review by jotamartos — CHANGES_REQUESTED** [review]

File: bitnami/rabbitmq/Chart.yaml [hunk 1]
```diff
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/rabbitmq
   - https://www.rabbitmq.com
-version: 11.0.4
+version: 11.0.5
```
Comment (jotamartos) [inline bitnami/rabbitmq/Chart.yaml:26]: Sorry for not pointing this out before, you need to bump the version to a new minor `11.1.0` as you are including new parameters to the chart.

Comment (cleverhu) [reply bitnami/rabbitmq/Chart.yaml:26]: Ok.I will update it later.

Comment (cleverhu) [reply bitnami/rabbitmq/Chart.yaml]: Ok.I will update it later.

**Review by jotamartos — APPROVED** [review]

**Pull request merged**